code/modules/mob/mob_helpers.dm

```diff
@@ -1127,4 +1127,50 @@ var/list/wierd_mobs_inclusive = list( /mob/living/simple_animal/construct,
 
 	else return null
 
+	
+#define POSESSIVE_PRONOUN	0
+#define POSESSIVE_ADJECTIVE	1
+#define REFLEXIVE			2
+#define SUBJECTIVE_PERSONAL	3
+#define OBJECTIVE_PERSONAL	4
+/mob/proc/get_pronoun(var/type)

```

Comment:
It should also utilize a switch() statement instead of if/elif x n/else. Because efficiency.